ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/HardFork/Combinator/Serialisation/Common.hs

```diff
@@ -371,6 +363,15 @@ data HardForkEncoderException where
   HardForkEncoderFutureEra :: SingleEraInfo blk -> HardForkEncoderException
 
   -- | HFC enabled, but we saw a value from a disabled era
+  --
+  -- This is only thrown by the Node-to-Client encoder. Two nodes' negotiated
```

Comment:
It is also thrown in the decoder: https://github.com/IntersectMBO/ouroboros-consensus/blob/cdf512c29091d5b66642b025085100a967b6fa3b/ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/HardFork/Combinator/Serialisation/SerialiseNodeToClient.hs#L108

Reply:
Done, thanks!